doc/sphinx/conf.py

```diff
@@ -41,12 +41,11 @@
               'sphinx.ext.autodoc',
               'sphinx.ext.todo',
               'sphinx.ext.autosummary',
-              'sphinx.ext.mathjax',
               'sphinxcontrib.doxylink',
+              'sphinxcontrib.katex',  # Use KaTeX because it's faster and the main site uses it
```

Comment:
Is this requirement documented somewhere?

Reply:
Yes, here: http://engr.uconn.edu/~weber/cantera/compiling/dependencies.html#sec-dependencies